source/compatibility.tex

```diff
@@ -2996,10 +2996,8 @@
 \end{example}
 
 \howwide
-This is fairly widely used but it is good
-programming practice to add the cast when assigning pointer-to-void to pointer-to-object.
-Some C translators will give a warning
-if the cast is not used.
+This is fairly widely used in C, but C++ requires a
+cast when assigning pointer-to-void to pointer-to-object.
```

Comment:
```suggestion
Common.
```
The second part of the sentence doesn't actually have to do with the `\howwide` element; it repeats information above.

I do think it's best to remove the value judgment of what "good practice" is; it's not the job of the C Annex to tell you what good/bad programing style is.

Reply:
Looking at it again, I think I'll withdraw this suggestion. "Common." makes sense, given that the section is about ISO C.

\<emily_litella\>Never mind.\</emily_litella>